### ISSUE TITLE
Add --format

### ISSUE THREAD

**Issue by erayerdin** [hugovk/pypistats#20]
Instead of defaulting markdown and providing extra `--json` argument, you can provide a `--format` argument default argument of which can be markdown.

```
# Instead of these
pypistats recent pillow  # default markdown output
pypistats recent pillow --json  # json output

# can be these
pypistats recent pillow  # markdown output
pypistats recent pillow --format markdown  # also markdown output
pypistats recent pillow --format json  # json output
pypistats recent pillow --format rst
pypistats recent pillow --format yaml
pypistats recent pillow --format html
```

And, as seen above, formats can be extended.

If you want, I can provide a PR to close this issue.

**Comment by hugovk** [comment]
Sounds good, I'd like to keep `--json` as a shortcut for `--format json`. Maybe handle that like these:

https://github.com/hugovk/pypistats/blob/46b6a6eb145837c76ffec642aa89686a99a60e05/pypistats/cli.py#L216-L219

pytablewriter is used for Markdown, and it can also do reStructuredText and HTML:

https://pytablewriter.readthedocs.io/en/latest/pages/examples/table_format/index.html

The original JSON data comes directly from the pypistats API. For Markdown, we only show the `data` part of the JSON, but for JSON the whole thing is returned.

https://github.com/hugovk/pypistats/blob/46b6a6eb145837c76ffec642aa89686a99a60e05/pypistats/__init__.py#L52

I guess we'd do the same for RST and HTML as MD, and YAML the same as JSON?

**Comment by erayerdin** [comment]
> I guess we'd do the same for RST and HTML as MD, and YAML the same as JSON?
For the sake of consistency, yes, we'd do the same.

 > pytablewriter is used for Markdown, and it can also do reStructuredText and HTML...
Well then, that's good news.

**Comment by hugovk** [comment]
Thanks, I've released pypistats 0.2.0.

**Comment by erayerdin** [comment]
No problem. I'm happy for the fact that I could contribute.
